UltiSnips/rust.snippets

```diff
@@ -4,12 +4,34 @@
 
 priority -50
 
-snippet fn "A function, optionally with arguments and return type." b
+snippet let "let variable declaration" b
+let ${1:name}${2:: ${3:type}} = ${4};
+endsnippet
+
+snippet letm "let mut variable declaration" b
+let mut ${1:name}${2:: ${3:type}} = ${4};
+endsnippet
+
+snippet fn "A function, optionally with arguments and return type."
```

Comment:
Removed the `b` option from this because it prevented use when defining a `pub fn`.